moveit_setup_assistant/src/widgets/configuration_files_widget.cpp

```diff
@@ -964,8 +964,18 @@ bool ConfigurationFilesWidget::checkGenFiles()
           ROS_WARN_STREAM("Editing in Setup Assistant conflicts with external editing of file " << file->file_name_);
 
         file->generate_ = false;  // do not overwrite by default
-        file->modified_ = true;
-        found_modified = true;
+
```

Comment:
Why do you need this change? As far as I understand, this enforces overwriting the file even if it has changes since last MSA generation.